### PR TITLE
XRT_INSTALL_DIR change makes mpd/msd not working

### PR DESCRIPTION
**Pull request by xuhz**

### CODE REVIEW

File: src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in [hunk 1]
```diff
@@ -6,7 +6,7 @@ ConditionDirectoryNotEmpty=/dev/xfpga
 
 [Service]
 Type=simple
-ExecStart=${XRT_INSTALL_DIR}/bin/mpd
+ExecStart=${CMAKE_INSTALL/PREFIX}/${XRT_INSTALL_DIR}/bin/mpd
```
Comment (stsoe) [inline src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in:9]: Please use ${CMAKE_INSTALL_PREFIX}/${XRT_INSTALL_BIN_DIR}/mpd

File: src/runtime_src/core/pcie/tools/cloud-daemon/msd.service.in [hunk 2]
```diff
@@ -6,7 +6,7 @@ ConditionDirectoryNotEmpty=/dev/xfpga
 
 [Service]
 Type=simple
-ExecStart=${XRT_INSTALL_DIR}/bin/msd
+ExecStart=${CMAKE_INSTALL_PREFIX}/${XRT_INSTALL_DIR}/bin/msd
```
Comment (stsoe) [inline src/runtime_src/core/pcie/tools/cloud-daemon/msd.service.in:9]: Ditto

**Review by stsoe — CHANGES_REQUESTED** [review]

File: src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in (hunk 1 above)
Comment (xuhz) [reply src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in]: changed.

File: src/runtime_src/core/pcie/tools/cloud-daemon/msd.service.in (hunk 2 above)
Comment (xuhz) [reply src/runtime_src/core/pcie/tools/cloud-daemon/msd.service.in]: changed

File: src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in [hunk 3]
```diff
@@ -6,7 +6,7 @@ ConditionDirectoryNotEmpty=/dev/xfpga
 
 [Service]
 Type=simple
-ExecStart=${XRT_INSTALL_DIR}/bin/mpd
+ExecStart=${CMAKE_INSTALL/PREFIX}/${XRT_INSTALL_BIN_DIR}/mpd
```
Comment (stsoe) [inline src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in:9]: CMAKE_INSTALL_PREFIX  :-)

**Review by stsoe — CHANGES_REQUESTED** [review]

File: src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in (hunk 3 above)
Comment (xuhz) [reply src/runtime_src/core/pcie/tools/cloud-daemon/mpd.service.in]: silly me. I also ran a build and happened just verifying the other one.
thanks for the catch
changed.

**Review by stsoe — APPROVED** [review]

**Comment by stsoe** [comment]
@maxzhen Given the imposed lock down, please merge if you deem it okay.

**Review by maxzhen — APPROVED** [review]
this is not impacting u30, so should be ok to merge.

**Pull request merged**